assets/src/domain/eventEditor/hooks/entityListCacheUpdates/test/useUpdatePriceTypeList.test.ts

```diff
@@ -59,7 +60,7 @@ describe('useUpdatePriceTypeList', () => {
 		);
 		await waitForUpdate({ timeout });
 
-		const cachedPriceTypeIds = cacheResult.current.map(({ id }) => id);
+		const cachedPriceTypeIds = getGuids(cacheResult.current);
```

Comment:
👍 great, we want to reuse logic as much as possible, even for tiny things like this (also think it reads better)